main.py

```
@@ -636,28 +645,24 @@ def create_reading_order(self):
             logging.info(dialog.textbox.text())
 
     def clicked_reading_order(self, id: int, name: str):
```

Ruff (0.15.12)

[error] Function argument `id` is shadowing a Python builtin

(A002)